src/Microsoft.DotNet.Watcher.Tools/Microsoft.DotNet.Watcher.Tools.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+  <Import Project="..\..\build\common.props" />
```

Comment:
> by default this is relative to the project file right?

Yes.

> I still think this should be a package.

Agreed. As per ongoing discussion in https://github.com/aspnet/specs/pull/49, I fully intend to move most of these common properties to a package.  When that happens, we can remove duplication from that file. That said, there is still a high chance we will still want a common.props file per repo for settings that truly are repo specific, such as `RepositoryUrl`.